controllers/api/constants.go

```diff
@@ -4,10 +4,17 @@ import (
 	"errors"
 )
 
-const ContentType = "application/vnd.api+json; charset=utf-8"
+const (
+	ContentType     = "application/vnd.api+json; charset=utf-8"
+	DefaultPageSize = 10
+)
 
 var (
-	ErrorInvalidUser        = errors.New("invalid user")
-	ErrorMissingAccessToken = errors.New("missing access token")
-	ErrorNotFoundUser       = errors.New("user not found")
+	DefaultOrderBy = []string{"created_at desc"}
+
+	ErrorInvalidUser           = errors.New("invalid user")
+	ErrorInvalidPayloaderType  = errors.New("invalid payloader type")
```

Comment:
Did you mean:

```suggestion
	ErrorInvalidPayloadType  = errors.New("invalid payload type")
```

Note sure what would be `payloader` 😅